build tests test/dblat1 and ctest/xdcblat1 fail on OpenIndiana SPARC

while building OpenBLAS-0.3.30 on OpenIndiana SPARC the following error happens in directory: test

OPENBLAS_NUM_THREADS=1 OMP_NUM_THREADS=1 ./dblat1
 Real BLAS Test Program Results
...
 Test of subprogram number  7            DNRM2 

Program received signal SIGBUS: Access to an undefined portion of a memory object.

Backtrace for this error:
/bin/sh: 181992: Abort(coredump)
make[1]: *** [Makefile:52: level1] Abort
make[1]: *** Waiting for unfinished jobs....

The second error happens in directory ctest:

OPENBLAS_NUM_THREADS=2 ./xdcblat1
 Real CBLAS Test Program Results
...

 Test of subprogram number  7         CBLAS_DNRM2    

Program received signal SIGBUS: Access to an undefined portion of a memory object.

Backtrace for this error:
/bin/sh: 186674: Abort(coredump)
make[1]: *** [Makefile:101: all1] Abort
make[1]: *** Waiting for unfinished jobs....

both Makefiles have been patched to not use these two tests, for details see:
https://github.com/OpenIndiana/oi-userland/pull/24841

--- a/kernel/sparc/KERNEL
+++ b/kernel/sparc/KERNEL
@@ -59,7 +59,8 @@ SNRM2KERNEL = snrm2.S
 endif
 
 ifndef DNRM2KERNEL
-DNRM2KERNEL = dnrm2.S
+#DNRM2KERNEL = dnrm2.S
+DNRM2KERNEL = ../arm/nrm2.c
 endif
 
 ifndef CNRM2KERNEL

